.travis.yml

```diff
@@ -1,10 +1,16 @@
-services:
-  - postgresql
-
 language: python
 
 python:
-  - 3.5
+- "3.5"
+
+services:
+  - postgresql
+
+env:
+  - TOX_ENV=django111
+  - TOX_ENV=django20
+  - TOX_ENV=django21
+  - TOX_ENV=django22
```

Comment:
These should be `TOXENV`; Travis isn't actually using tox because the Makefile condition doesn't match the environment variable name.  I was wondering what actually installs tox since I didn't see it in the requirements files; we probably want a new `travis.in` and `travis.txt` for this.